### PR TITLE
Add join table

### PR DESCRIPTION
**Pull request by didierganthier**
## Checklist
- [x] I have read this repository's contribution guidelines.
- [x] My pull request has a descriptive title (not a vague title like Update index.MD)
- [x] My pull request targets the main branch of this [project](https://github.com/didierganthier/vet-clinic-database).
- [x] I added relevant documentation for all the files I changed

## Screenshot of the results of the queries
Screenshot 2022-12-08 at 12 05 32

## Screenshot of the results of the queries
Screenshot 2022-12-08 at 12 05 58

## Screenshot of the results of the queries
Screenshot 2022-12-08 at 12 06 28

## Screenshot of the results of the queries
Screenshot 2022-12-08 at 12 06 44

This PR creates tables, inserts data, and makes some queries

 ## What changes have been made?
- [x] [Create `vets` table](https://github.com/didierganthier/vet-clinic-database/commit/7b6392daa780d09e473b73958cf238839dddf4a2)
- [x] [Create `JOIN` tables](https://github.com/didierganthier/vet-clinic-database/commit/75b0bf4899b05517a8e42b7339c623d98850b1a1)
- [x] [Insert `vets` and `species` data](https://github.com/didierganthier/vet-clinic-database/commit/d46d2ca245695b43cf426118ea3018d912010c72)
- [x] [Insert `visits` data](https://github.com/didierganthier/vet-clinic-database/commit/d0599d0abe22df1e7f30d48b22f7c26a186c1b11)
- [x] [Write queries to answer questions in `queries.sql`](https://github.com/didierganthier/vet-clinic-database/commit/e2be3fc51547790bf13d70c5433717a46dd81ad0)

The author of these changes is the owner of the repo @didierganthier

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -86,3 +86,70 @@ GROUP BY owners.full_name
...
+WHERE specializations.species_id IS NULL;
+
+SELECT COUNT(visits.animal_id), animals.name
+FROM visits
+JOIN vets ON visits.vet_id = vets.id
+JOIN animals ON visits.animal_id = animals.id
+WHERE vets.name = 'Maisy Smith'
+GROUP by animals.name
+ORDER by COUNT(animals.name) DESC
+LIMIT 1 ;
```
Comment (ShadyShawkat) [inline queries.sql]: - [ ] As per the [required query](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_join_table.md#:~:text=What%20specialty%20should%20Maisy%20Smith%20consider%20getting%3F%20Look%20for%20the%20species%20she%20gets%20the%20most.), please select the specialty   'Maisy Smith' gets the most and not the animal name.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @didierganthier, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (didierganthier) [reply queries.sql]: Thank you @ShadyShawkat I will handle that 🙃

**Review by descholar-ceo — APPROVED** [review]
Hi @didierganthier 🤝 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
👍🏽 Implemented all of the requirements ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Congratulations! 🎉

**Pull request merged**